src/admin-views/editor/panel/ticket.php

```diff
@@ -189,7 +216,132 @@ class="ticket_field ticket_provider"
 				 */
 				do_action( 'tribe_events_tickets_metabox_edit_main', $post_id, $ticket_id ); ?>
 			</section>
-
+			<div class="input_block">
+				<label class="ticket_form_label ticket_form_left" for="ticket_description"><?php esc_html_e( 'Description:', 'event-tickets' ); ?></label>
+				<textarea
+						rows="5"
+						cols="40"
+						name="ticket_description"
+						class="ticket_field ticket_form_right"
+						id="ticket_description"
+				><?php echo esc_textarea( $ticket ? $ticket->description : null ) ?></textarea>
+				<div class="input_block">
+					<label class="tribe_soft_note">
+						<input
+								type="checkbox"
+								id="tribe_tickets_show_description"
+								name="ticket_show_description"
+								value="1"
+								class="ticket_field ticket_form_left"
+								<?php checked( true, $ticket ? $ticket->show_description : true ); ?>
+						>
+						<?php
+						echo esc_html( sprintf(
+								__( 'Show description on front end %s form.', 'event-tickets' ),
+								tribe_get_ticket_label_singular_lowercase( 'default_ticket_provider' )
+						) );
+						?>
+					</label>
+				</div>
+			</div>
+			<div class="input_block">
+				<label class="ticket_form_label ticket_form_left" for="ticket_start_date"><?php esc_html_e( 'Start sale:', 'event-tickets' ); ?></label>
+				<div class="ticket_form_right">
+					<input
+							autocomplete="off"
+							type="text"
+							class="tribe-datepicker tribe-field-start_date ticket_field"
+							name="ticket_start_date"
+							id="ticket_start_date"
+							value="<?php echo esc_attr( $ticket ? $start_date : null ); ?>"
+							data-validation-type="datepicker"
+							data-validation-is-less-or-equal-to="#ticket_end_date"
+							data-validation-error="<?php echo esc_attr( json_encode( $start_date_errors ) ) ?>"
```

Comment:
* :no_entry_sign: Inline PHP statement must end with a semicolon
* :warning: json_encode() is discouraged. Use wp_json_encode() instead.

<sub>hash: 4a40cdae7f105799aba0718264a21e58</sub>